src/components/FAQ.tsx

```diff
@@ -0,0 +1,74 @@
+import { useState } from 'react';
+
+const FAQSection = () => {
+  const [activeIndex, setActiveIndex] = useState<number | null>(null);
```

Comment:
can you use activeIndex value as -1, instead of using null type. It will be simpler this way.